agent/tools/dispatch_sprints_test.go

```
@@ -0,0 +1,461 @@
+		Usage:   llm.Usage{InputTokens: 80, OutputTokens: 5},
+	}
+}
+
+func writeJSONL(t *testing.T, dir string, lines []dispatchEntry) string {
+	t.Helper()
+	path := filepath.Join(dir, "plan.jsonl")
+	f, err := os.Create(path)
+	if err != nil {
+		t.Fatalf("create plan: %v", err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `f.Close` is not checked

(errcheck)